README.md

```diff
@@ -574,27 +575,116 @@ Input:
 
 ```json
 {
-    "Type": "Transcribe",
-    "LanguageCode": "en-US",
-    "MediaFormat": "mp3",
-    "Destination": {
-        "Mode": "AWS/S3",
-        "BucketName": "myBucket",
-        "ObjectKey": "myTranscript.json"
-    }
+  "Type": "Transcribe",
+  "LanguageCode": "en-US",
+  "MediaFormat": "mp3",
+  "Destination": {
+    "Mode": "AWS/S3",
+    "BucketName": "myBucket",
+    "ObjectKey": "myTranscript.json"
+  }
 }
 ```
 
 Output:
 
 ```json
 {
-    "Task": "Transcribe",
+  "Task": "Transcribe",
+  "BucketName": "myBucket",
+  "ObjectKey": "myTranscript.json"
+}
+```
+
+### WAV Wrap
+
+`WavWrap` tasks create a [WAV](https://en.wikipedia.org/wiki/WAV) file from an audio artifact, and apply data to specific chunks of the WAV wrapper.
+
+It accepts MPEG audio files, and any of the chunks supported by [prx-wavefile](https://github.com/PRX/prx-wavefile) including all Broadcast Wave Format chunks and `cart` chunks.
+
+`prx-wavefile` will attempt to set the `fmt`, `mext`, `bext`, `fact`, and `data` chunks from the source file, e.g. analyzing the mpeg audio to set the `fmt` sample rate, bit rate, and number of channels.
+
+The `cart` chunk will not be set unless at all unless it is passed in to the input `Task.Chunks` array, as in the example below.
+
+The Output includes a `WavfileChunks` array with the attributes set on the `prx-wavefile` chunks.
+
+Input:
+
+```json
+{
+  "Type": "WavWrap",
+  "Destination": {
+    "Mode": "AWS/S3",
     "BucketName": "myBucket",
     "ObjectKey": "myTranscript.json"
+  },
+  "Chunks": [
+    {
+      "ChunkId": "cart",
+      "Version": "0101",
+      "CutId": "12345",
+      "Title": "Title",
+      "Artist": "Artist",
+      "StartDate": "2020/01/01",
+      "StartTime": "00:00:00",
+      "EndDate": "2020/01/14",
+      "EndTime": "00:00:00",
+      "ProducerAppId": "PRX",
+      "ProducerAppVersion": "3.0"
+    }
+  ]
+}
+```
+
+Output:
+
+````json
+{
+  "Task": "WavWrap",
+  "Mode": "AWS/S3",
+  "BucketName": "myBucket",
+  "ObjectKey": "myTranscript.json",
+  "Time": "2012-12-21T12:34:56Z",
+  "Timestamp": 1356093296.123,
+  "WavefileChunks": [
+    {
+      "chunkId": "cart",
+      "chunkSize": 2048,
+      "version": "0101",
+      "title": "Title",
+      "artist": "Artist",
+      "cutId": "12345",
+      "clientId": "",
+      "category": "",
+      "classification": "",
+      "outCue": "",
+      "startDate": "2020/01/01",
+      "startTime": "00:00:00",
+      "endDate": "2020/01/14",
+      "endTime": "10:00:00",
+      "producerAppId": "PRX",
+      "producerAppVersion": "3.0",
+      "userDef": "",
+      "levelReference": 0,
+      "postTimer": [],
+      "reserved": "",
+      "url": "",
+      "tagText": ""
+    }
+  ]
 }
 ```
 
+#### AWS/S3
```

Comment:
There are a couple references in this section to images, because I copied it from that task, which should be updated.

Reply:
fixed.